react-todo-app/src/TodoContainer/TodoContainer.js

```diff
@@ -0,0 +1,355 @@
+    try {
+      setIsLoading(true);
+      const data = await requestEditDescription(
+        id,
+        newEditDescription,
+        tableName
+      );
+      const editedTodoList = todoList.map((todoItem) => {
+        if (todoItem.id === data.id) {
+          return {
+            ...todoItem,
+            fields: {
+              ...todoItem.fields,
+              Title: data.fields.Title,
+              Description: data.fields.Description,
+              Done: data.fields.Done,
+            },
+          };
+        }
+        return todoItem;
+      });
+      setTodoList(editedTodoList);
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const handleDescription = (id) => {
+    setItemDescription(id);
+    setDrawerVisible(true);
+  };
+
+  return (
+    <Layout
+      style={{
+        display: "flex",
+        justifyContent: "flex-start",
+        alignItems: "center",
+        height: "100vh",
+      }}
+    >
+      <Header
+        className={style.header}
+        mode="block"
+        style={{
+          backgroundColor: "var(--primary-color)",
+          padding: "0 2rem",
+          width: "100%",
+          height: "3rem",
+          display: "flex",
+          alignItems: "flex-start",
+          justifyContent: "flex-start",
+        }}
+      >
+        <div className="logo">
+          <p style={{ color: "#fff", fontSize: "20px" }}>LaserMind</p>
+        </div>
+      </Header>
+      <Content
+        style={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          height: "100vh",
+        }}
+      >
+        <div className={style.container}>
+          <div className={style.todo_list}>
+            {errorMessage && <p className={style.error}>{errorMessage}</p>}{" "}
+            {/* User error message */}
+            <Tooltip title="Enter the Matrix!">
+              <h1
+                className={style.banner}
+                onClick={() => handleImageClick(RedPillImg)}
+              >
+                To Do or Not To Do !
+              </h1>
+            </Tooltip>
+            <Tooltip title="Click Me!">
+              <img
+                src={image}
+                alt="{table_name}"
+                className={style.image_link}
+                onClick={() => handleImageClick(image)}
+              />
+            </Tooltip>
+            <h5 className={style.table_name}>{tableName}</h5>
+            <AddTodoForm
+              onAddTodo={addTodo}
+              todoList={todoList}
+              tableName={tableName}
+            />
+            <div
+              style={{
+                display: "flex",
+                flexDirection: "row",
+                alignItems: "center",
+                width: "100%",
+                justifyContent: "space-around",
+              }}
+            >
+              <div>
+                <Tooltip title="Sort by Title">
```

Comment:
The sorting is super slick, it's fast and easy. It might be easier for users to understand how the results are sorted, which field and which order, if you move the sort options into a dropdown select like this 
![image](https://user-images.githubusercontent.com/6400323/233337434-54a835f5-664f-40ad-ac6e-1d7cc578c6ea.png)

This way the selected option will display and the user will know how the results are sorted. What do you think of this?

Reply:
And more thoughts on sort actually: I found I could sort the todos, then add a todo that was sorted onto a different page and so couldn't actually see the todo I just created.

I'm wondering if it's necessary for the user to be able to sort by different fields. What was your intention with this feature? One way to address the behavior I found is to remove the different sort options and to always sort by create time descending.